.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout commit
+      uses: actions/checkout@v1
+
+    - name: Checkout submodules
+      run: git submodule update --init --recursive
+
+    - name: Cache Cargo
+      uses: actions/cache@v1
+      with:
+        path: ~/.cargo
+        key: ${{ runner.os }}-cargo
+    
+    - name: Cache binaryen
+      uses: actions/cache@v1
+      with:
+        path: ~/binaryen
+        key: ${{ runner.os }}-binaryen
+    
+    - name: Cache target
+      uses: actions/cache@v1
+      with:
+        path: livesplit-core/target
+        key: ${{ runner.os }}-target
+
+    - name: Install Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
I'd prefer not to specify a version at all / always choose the latest Node.

Reply:
The default version for `actions/setup-node` is Node 10, but I think we want to be on Node 12 (since it's the latest LTS release). I don't think this action lets you specify "latest LTS" any other way besides `12.x`.

Reply:
Is there an issue tracking this there? Or should we maybe open one?

Reply:
https://github.com/actions/setup-node/issues/26

Reply:
Can you mark it with a comment `FIXME: https://github.com/actions/setup-node/issues/26` so we remember checking this.